data/styles/default/light/colors.mapcss

```diff
@@ -52,6 +52,7 @@
 
 @green0: #CFD5A8;
 @green1: #C6CF93;
+@green7: #C1CC89;
```

Comment:
It would be better to keep enumeration follow color darkness/intensity.
Call it `@green1b` maybe :)